apprentice_teacher.rb

```diff
@@ -1,29 +1,15 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative 'teacher'
 
-  def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
-    @target_raise = 800
-    @phase = 3
-  end
+class ApprenticeTeacher < Teacher
 
-  def offer_high_five
-    "High five!"
-  end
+  attr_reader :raise, :rating
 
-  def set_phase(num)
-    @phase = num
-    "Cool, I've always wanted to teach phase #{num}!"
-  end
 
-  def teach_stuff
-    response = ""
-    response += "Listen, class, this is how everything works. "
-    response += "*drops crazy knowledge bomb* "
-    response += "... You're welcome."
-    response
+  def initialize(options={})
+    super
+    @phase = 3
+    @rating = 80
+
   end
 
   def salary=(new_salary)
```

Comment:
You're redefining a bunch of these that are defined in the superclass already. Take them out here.